mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx

```diff
@@ -0,0 +1,260 @@
+import React, { Dispatch, useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { tailwind } from '@tailwind'
+import { StackScreenProps } from '@react-navigation/stack'
+import { NavigationProp, useNavigation } from '@react-navigation/native'
+import BigNumber from 'bignumber.js'
+import { RootState } from '@store'
+import { firstTransactionSelector, hasTxQueued as hasBroadcastQueued } from '@store/ocean'
+import { translate } from '@translations'
+import { hasTxQueued, transactionQueue } from '@store/transaction_queue'
+import { CompositeSwap, CTransactionSegWit } from '@defichain/jellyfish-transaction'
+import { PoolPairData } from '@defichain/whale-api-client/dist/api/poolpairs'
+import { WhaleWalletAccount } from '@defichain/whale-api-wallet'
+import { ConversionMode, dfiConversionCrafter } from '@api/transaction/dfi_converter'
+import { onTransactionBroadcast } from '@api/transaction/transaction_commands'
+import { NativeLoggingProps, useLogger } from '@shared-contexts/NativeLoggingProvider'
+import { ThemedIcon, ThemedScrollView, ThemedSectionTitle, ThemedView } from '@components/themed'
+import { TextRow } from '@components/TextRow'
+import { NumberRow } from '@components/NumberRow'
+import { FeeInfoRow } from '@components/FeeInfoRow'
+import { PricesSection } from './components/PricesSection'
+import { TransactionResultsRow } from '@components/TransactionResultsRow'
+import { SubmitButtonGroup } from '@components/SubmitButtonGroup'
+import { SummaryTitle } from '@components/SummaryTitle'
+import { getNativeIcon } from '@components/icons/assets'
+import { ConversionTag } from '@components/ConversionTag'
+import { View } from '@components'
+import { InfoText } from '@components/InfoText'
+import { DerivedTokenState } from './CompositeSwapScreen'
+import { DexParamList } from '../DexNavigator'
+
+type Props = StackScreenProps<DexParamList, 'ConfirmCompositeSwapScreen'>
+export interface CompositeSwapForm {
+  tokenFrom: DerivedTokenState
+  tokenTo: DerivedTokenState
+  amountFrom: BigNumber
+  amountTo: BigNumber
+}
+
+export function ConfirmCompositeSwapScreen ({ route }: Props): JSX.Element {
```

Comment:
hmm does it make sense to have normal pool swap composite swap as 2 different pages? (or am I looked at outdated UI design?)

Reply:
@ivan-zynesis yes, we will remove the pool swap page and redirect to composite swap once it's done. cc: @thedoublejay

Reply:
Yup, once this Composite swap logic has been completed, we will migrate the other (pool swap pages) to use this screen and remove the old pool swap pages.